Makefile

```
@@ -1,13 +1,13 @@
 all: test
 
 flakes:
-	pyflakes search/*.py
+	pyflakes spacegraphcats/*.py
```

note, this only looks at spacegraphcats/*.py, not the stuff under that directory. Would probably need to do
```suggestion
 	pyflakes $(find spacegraphcats -name \*.py)
```

---

Looks like `pyflakes spacegraphcats` works as well.